CONTRIBUTING.rst

```diff
@@ -21,3 +21,21 @@ ease-of-use for the user interface as well as naming suggestions for the
 various objects and options that users interact with.
 While stabilizing the interface will take precedence for the time being,
 new feature requests are always welcome!
+
+Development Details
+-------------------
+
+* Begin development work on ``sciform`` by forking and installing the
+  `git repository <https://github.com/jagerber48/sciform>`_ and
+  installing the development dependencies in editable mode with::
+
+     python -m pip install -e .[dev]
```

Comment:
```suggestion
     python -m pip install -e ".[dev]"
```

Trying to install optional dependencies without quotes gives "No matches found" errors on some machines. I believe it is if people are using zsh, but don't quote me on that (no pun intended 😂). 

Using quotes is a pip mechanism for optional dependencies, and should be recognized on all machines.